packages/tokens/skip-link-tokens/tokens.json

```diff
@@ -66,12 +80,19 @@
         },
         "$type": "dimension"
       },
-      "text-decoration": {
+      "text-decoration-thickness": {
         "$extensions": {
-          "nl.nldesignsystem.css-property-syntax": ["inherit", "none", "underline"],
```

Comment:
Klopt het dat we geen text-decoration-line meer willen supporten? Iedereen krijgt gewoon de default?

Reply:
Yes, er is geen situatie dat we de skip link er niet uit willen laten zien als een link. Ik ben ook afgestapt van de `ButtonFace` kleuren voor forced colors mode, zoals dat bij Utrecht was.